tests/puppeteer/pages/commonPage.js

```diff
@@ -154,14 +154,12 @@ module.exports = class CommonPage {
   /**
    * Check attribute value
    * @param selector, element to check
-   * @param attribute, attribute to test
+   * @param attribute, attribute to get
    * @param textToCheckWith, text to check with
    * @return promise, throw an error if element does not exist or attribute value is not correct
    */
   async checkAttributeValue(selector, attribute, textToCheckWith) {
```

Comment:
Maybe this function is not useful because we are checking the values in the tests.

Reply:
Nice catch, deleted :+1: